### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by eldipa**
Minor changes plus a workaround for the bug in `joern` that it cannot process `try/catch` blocks.

### CODE REVIEW

**Comment by eldipa** [comment]
Ugly side effect of the fix: it shifts all the lines numbers so we need to compensate that before merging the pr

File: fefobot.sh [hunk 1]
```diff
@@ -66,16 +66,19 @@ else
...
 # delete any non source code file recursively
 find . -type f -not -name '*.cpp' -a -type f -not -name '*.h' -a -type f -not -name 'Makefile' -delete
 # delete directories that may have emptied after deleting files
 find . -type d -empty -delete
+# inject some code into each source code
+find . -type f \( -name '*.cpp' -o -name '*.h' \) -a -not -name REPO_HEAD.h -exec ../inject_source_code.sh {} \;
 
-joern --nocolors < ../joern_commands.scala > /dev/null
+echo "Running Joern"
+TERM=dumb /home/user/bin2/joern/joern-cli/joern --nocolors < ../joern_commands.scala > /dev/null
```
Comment (leogm99) [inline fefobot.sh:81]: we should probably search the location of the joern executable (maybe using `find`)

File: inject_source_code.sh
Comment (leogm99) [inline inject_source_code.sh:1]: nice! way cleaner than using `sed`

**Comment by leogm99** [comment]
> Ugly side effect of the fix: it shifts all the lines numbers so we need to compensate that before merging the pr

yes. I think we can fix this by subtracting the number of lines injected from each issue

File: fefobot.sh (hunk 1 above)
Comment (eldipa) [reply fefobot.sh]: ups!! I totally forgot that I hardcoded the path to test different version of `joern`. I will revert this.

**Review by leogm99 — APPROVED** [review]

**Pull request merged**